ui/vty-app/Glue.hs

```diff
@@ -187,7 +200,11 @@ knitFaceToUI UiFace{..} KnitFace{..} putPass =
             Knit.ValueHash h -> Right $ pretty h
             _ -> Left "Unrecognized return value"
       UiFee{} ->
-        Just . UiFeeCommandResult . UiFeeCommandSuccess $ "not implemented"
+        Just . UiFeeCommandResult . either UiFeeCommandFailure UiFeeCommandSuccess $
+          fromResult result >>= fromValue >>= \case
+            Knit.ValueCoin c -> Right $ pretty c
```

Comment:
IIRC there is some function to display a `Coin` value in Ariadne UI.
@maksbotan I don't remember how it's called and where it's defined, could you help please?

Reply:
See https://github.com/serokell/ariadne/blob/917e2e17df25793bd7602db6c05ba0f35f9fc5ab/ui/vty-app/Glue.hs#L399